src/librustc_session/options.rs

```diff
@@ -924,7 +924,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         parse_switch_with_opt_path, [UNTRACKED],
         "run the self profiler and output the raw event data"),
     self_profile_events: Option<Vec<String>> = (None, parse_opt_comma_list, [UNTRACKED],
-        "specifies which kinds of events get recorded by the self profiler"),
+        "specifies which kinds of events get recorded by the self profiler;
+        for example: `-Z self-profile-events=default,query-keys`"),
```

Comment:
can we not list all event types that is possible to record?

Reply:
Is there a good way of keeping the list in sync with the one in `profiling.rs`?

Reply:
I'm not aware of any good way of doing that. I'm not oppressed to listing all the options here though because we change them pretty rarely and a comment here and in `profiling.rs` will probably keep it in sync 99% of the time.

Reply:
Sounds good to me.